pkg/daemon/ceph/osd/volume.go

```diff
@@ -116,6 +135,19 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 	}
 
 	if configured > 0 {
+		// Reporting
+		reportArgs := append(batchArgs, []string{
+			"--report",
+			"--format=json",
+		}...)
+		if err := context.Executor.ExecuteCommand(false, "", cephVolumeCmd, reportArgs...); err != nil {
```

Comment:
This will simply print the results to the log? Would it be more readable in the log if it wasn't json?

Reply:
It depends, personally I like the json format better but sure we can use the default format.